go/store/blobstore/gcs.go

```diff
@@ -55,11 +63,12 @@ func (bs *GCSBlobstore) Exists(ctx context.Context, key string) (bool, error) {
 // Get retrieves an io.reader for the portion of a blob specified by br along with
 // its version
 func (bs *GCSBlobstore) Get(ctx context.Context, key string, br BlobRange) (io.ReadCloser, string, error) {
-	oh := bs.bucket.Object(bs.prefix + key)
+	absKey := path.Join(bs.prefix, key)
+	oh := bs.bucket.Object(absKey)
 	attrs, err := oh.Attrs(ctx)
 
 	if err == storage.ErrObjectNotExist {
-		return nil, "", NotFound{key}
+		return nil, "", NotFound{"gs://" + path.Join(bs.bucketName, absKey)}
```

Comment:
gcs://?

Reply:
Didn't you work at google?  `gs://` is the scheme used to refer to files in GCS.